test/unit-test/framework/mock_hidapi.c

```diff
@@ -97,7 +97,10 @@ static void _hid_open(void)
     pthread_mutex_init(&mutex, NULL);
 }
 
-hid_device* hid_open(unsigned short vid, unsigned short pid, const wchar_t* path)
+hid_device* hid_open(
```

Comment:
looks unused to me -> delete whole function?

Reply:
ack